superset/utils/core.py

```diff
@@ -340,6 +340,8 @@ def base_json_conv(obj):
         return str(obj)
     elif isinstance(obj, timedelta):
         return str(obj)
+    elif isinstance(obj, memoryview):
+        return str(obj.tobytes(), 'utf8')
```

Comment:
Note in Python 3 the default encoding for [`str`](https://docs.python.org/3/library/stdtypes.html#str) is `utf-8`.

Reply:
On first iteration I tried calling `str` without the encoding, but that showed `b'abc'` in SQL Lab. A quick google turned up https://stackoverflow.com/questions/606191/convert-bytes-to-a-string , and after adding `'utf8'` to `str` the string rendered correctly. My current dev env is running py 3.7.2.

Reply:
This behavior is explained in the documentation for [`str`](https://docs.python.org/3/library/stdtypes.html#str) and in your case you do need (as you have) to explicitly define the encoding even if it's the default. Obviously not the clearest logic.

Reply:
Thanks @john-bodley for the link, this functionality is now clearer to me. In retrospect calling `bytes.decode('utf-8')` might have been more intuitive.